pkg/api/v1/types.go

```diff
@@ -1068,6 +1068,12 @@ type KubeVirtConditionType string
 
 // These are the valid KubeVirt condition types
 const (
-	// Whether the deployment or deletion was successful
-	KubeVirtConditionSynchronized KubeVirtConditionType = "KubeVirtSynchronized"
+	// Whether the deployment or deletion was successful (only used if false)
+	KubeVirtConditionSynchronized KubeVirtConditionType = "Synchronized"
+	// Whether all resources were created
+	KubeVirtConditionCreated KubeVirtConditionType = "Created"
+	// Whether KubeVirt components are ready
+	KubeVirtConditionReady KubeVirtConditionType = "Ready"
```

Comment:
"Ready" condition here doesn't appear to be implemented.

Reply:
Correct, will be implemented in a follow up. Can remove it for this PR